src/server/node.rs

```diff
@@ -316,6 +319,7 @@ where
         Err(box_err!("check cluster bootstrapped failed"))
     }
 
+    #[allow(too_many_arguments)]
```

Comment:
Not allowed. :)

Reply:
https://github.com/pingcap/tikv/blob/ba174ad54c8e2ff9d45979a4f6a65ca01901308a/src/coprocessor/codec/mysql/time.rs#L43

Refactor this function to reduce arguments will make this signature less readable.